source/plugins/projects/projects/index.js

```diff
@@ -1329,7 +1329,7 @@ let projects = {
 	 * 
 	 * @example
 	 * 
-	 * saveSpecialFile('MyNewProject', 'SpecialFileName', [1, 2, 3]);
+	 * saveSpecialFile(project, 'SpecialFileName', [1, 2, 3]);
```

Comment:
As most of the data written in a file will be strings, I suggest having the example with Buffer.from ('.....')